mne/io/eeglab/eeglab.py

```diff
@@ -244,7 +257,7 @@ class RawEEGLAB(_BaseRaw):
     """
     @verbose
     def __init__(self, input_fname, montage, preload=False, eog=(),
-                 verbose=None):
+                 event_id=dict(), verbose=None):
```

Comment:
I'll look at this later ... need to rush. If you can simplify it would be great. This code is currently not very readable.